packages/dds/legacy-dds/src/signal/sharedSignalFactory.ts

```diff
@@ -9,16 +9,17 @@ import type {
 	IChannelFactory,
 	IChannelServices,
 } from "@fluidframework/datastore-definitions/internal";
+import { createSharedObjectKind } from "@fluidframework/shared-object-base/internal";
 
 import { pkgVersion } from "../packageVersion.js";
 
 import type { ISharedSignal } from "./interfaces.js";
-import { SharedSignal } from "./sharedSignal.js";
+import { SharedSignal as SharedSignalClass } from "./sharedSignal.js";
```

Comment:
rather than renaming at import, it would better to just rename the thing, otherwise the code is a bit confusing.

Reply:
+1 this